lib/client/cluster_client.go

```diff
@@ -84,6 +84,20 @@ func (c *ClusterClient) Close() error {
 	return trace.NewAggregate(c.AuthClient.Close(), c.ProxyClient.Close())
 }
 
+// ceremonyFailedErr indicates that the mfa ceremony was attempted unsuccessfully.
```

Comment:
```suggestion
// ceremonyFailedErr indicates failed mfa ceremony.
```
is that the same thing? ignore otherwise.